tests/test_cad_to_post_wtm.py

```diff
@@ -4,8 +4,8 @@
 This test covers the setup and solution of a three-dimensional
```

Comment:
Generally, we use 3D instead of writing out three-dimensional (given that our audience is well aware of the term).